.github/workflows/build-and-test.yaml

```diff
@@ -593,7 +602,7 @@ jobs:
       id: cache
       with:
         path: 'build/tests/**/*.beam'
-        key: ${{ matrix.otp }}-${{ hashFiles('**/build-and-test.yaml', 'tests/**/*.{erl,hrl,ex}') }}-${{ matrix.jit_target_arch }}
+        key: ${{ matrix.otp }}-${{ hashFiles('**/build-and-test.yaml', 'tests/**/*.erl', 'tests/**/*.hrl', 'tests/**/*.ex') }}-${{ matrix.jit_target_arch }}
```

Comment:
hmm, this should ultimately result in the same behavior over time eg. I believe `'tests/**/*.{erl,hrl,ex}'` is correct syntax - this obviously makes a new key and cache...

But let's try this for now..

Reply:
oh, wow - definitely go with the fix, `'tests/**/*.{erl,hrl,ex}'` is a wrong pattern it seems, apologies.